app/helpers/me/notices_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Me::NoticesHelper
+  def dissmiss_all_notices_button
+    link_to_remote_with_confirm(:dismiss_all_notices.t,
```

Comment:
But modal is made by another Prototype plugin, isn't it?
If we migrate into jQuery in future we may want to use Bootstrap's modals?
On Jul 11, 2015 5:01 PM, "azul" <notifications@github.com> wrote:

> In app/helpers/me/notices_helper.rb
> <https://github.com/riseuplabs/crabgrass-core/pull/269#discussion_r34412877>
> :
>
> > @@ -0,0 +1,9 @@
> > +module Me::NoticesHelper
> > +  def dissmiss_all_notices_button
> > +    link_to_remote_with_confirm(:dismiss_all_notices.t,
>
> hmm then we should make it so the confirm option of link_to also uses the
> nice modal version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/riseuplabs/crabgrass-core/pull/269/files#r34412877>.
>